scripts/mkcloud

```diff
@@ -774,9 +776,59 @@ function testsetup()
     onadmin testsetup
     local ret=$?
     $scp root@$adminip:tempest.log "$artifacts_dir/tempest.log"
+    $scp root@$adminip:tempest.subunit.log "$artifacts_dir/tempest.subunit.log"
```

Comment:
that doesn't work. you need to first copy it there (or copy it from the right place):

https://github.com/aplanas/automation/blob/master/scripts/qa_crowbarsetup.sh#L2652